lib/i18n/payment-method-provider-type.js

```diff
@@ -49,6 +54,6 @@ export const i18nPaymentMethodProviderType = (intl, providerType) => {
   } else if (i18nTypes[providerType]) {
     return intl.formatMessage(i18nTypes[providerType], { service: 'Open Collective' });
   } else {
-    return providerType;
+    return intl.formatMessage(i18nTypes[providerType], { providerType });
```

Comment:
Why change the default here? This change will crash the page rather than changing the default value if we receive an unknown one.

Reply:
I see your point and yes I agree. The reason I made this change was because the value of `providerType` is `"ADDED_FUNDS"` which is a hardcoded string. What we want to display is the translated message, i.e `"Added Funds"` which is not happening. I looked into the code but couldn't find the part where we're updating `providerType`.

Reply:
If we arrive to this `else`, it means there is no message for `providerType`. Thus calling this function will do:

```es6
intl.formatMessage(undefined, { providerType: 'Whatever' })
```

Which will crash the page.

Reply:
Right, I agree with you. If you see my comment above I'm trying to say that `providerType="ADDED_FUNDS"` which is hardcoded and that's why it's not getting translated.